spec/services/search_service_spec.rb

```diff
@@ -441,16 +445,16 @@
         }.ignore_extra_keys!
       end
 
-      it 'returns goods code if search date falls within validity period' do
-        result = TimeMachine.at('2005-01-01') { described_class.new(data_serializer, q: 'water').to_json }
+      context 'with search date falls within validity period returns goods code' do
```

Comment:
Small thing but I think you don't want your expectation here
```suggestion
      context 'with search date falls within validity period' do
```

Reply:
Yeah i thought these phrases could be improved